sdk/communication/azure-communication-chat/README.md

```diff
@@ -15,7 +15,7 @@ Azure Communication Chat contains the APIs used in chat applications for Azure C
 - A deployed Communication Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) or the [Azure PowerShell](https://docs.microsoft.com/powershell/module/az.communication/new-azcommunicationservice) to set it up.
 
 ### Versions available
-The current version of this library is **2.0.1**.
+The current version of this library is **2.1.0-beta.1**.
```

Comment:
Normally we want to keep the version mentioned in the README here pointing to the latest stable, so `2.0.1` in this case. We can update to a new version when we are about to release it.
```suggestion
The current version of this library is **2.0.1**.
```